### DISCUSSION TITLE
Achievement Quests

### DISCUSSION THREAD

**Discussion by julkasalsa** [RetroAchievements/RAWeb#667] [Ideas]
Alow admins to compose sets of achievements from different games which, after completion could give unique site award, plus optional other benefits for the player. Quests shlould be designed collectively by the community, but only admins should be allowed to edit them.

**Comment by meleu** [comment]
Didn't get what you mean. Could you provide an example?

**Comment by kdecks** [comment]
@meleu Essentially a set that is made up of from a collection of achievements from other sets.

I really like this concept. I also think that it would be fine as a developer permission. If we saw poor usage we could shut it down individually.

**Comment by julkasalsa** [comment]
I don't want us to have too many of those quests, becasue then they would be less attractive. I'm sure a lot of developers would have many ideas for those. I want to use this idea to make obscure games and sets more visible.

**Comment by DrPixelz** [comment]
I really love this idea - one of my favorite parts of playing stuff on Kongregate a few years ago was their quest system which basically works the same way. An example is [here](https://www.kongregate.com/badge_quests/pixel-pack-quest). They also have "Best Of" quests for each year where they acknowledge some of the best games of a particular year. I think we could do something along those lines but tie it into that Developer Awards idea you had, kdecks - just basically have a "quest" involving ~10 different achievements from some of the best sets of a particular year (with the first one being 2017).

**Comment by linusonline** [comment]
This sounds awesome. Can see a few use cases off the bat:
- Quests for completing/mastering a whole game series (mario, zelda, metroid, ff, etc).
- Quests making obscure games more visible
- Quests for similar cheevos from different games in one genre, like speed records for racing games, secret discovery for exploration games, collection completion for RPGs, etc.
- Quests for completing all games on one system released in the same year.

**Comment by Jamiras** [comment]
Achievement based quests can be done using the event framework. 

Quests targeting game completion will require additional work, and will be tracked by #1415.
